docs/guide/yaml/example_yaml/test-app-with-enrichers-slightly-simpler.yaml

```diff
@@ -0,0 +1,57 @@
+#
+# example showing how enrichers can be set 
```

Comment:
I'd prefer a file name that is more descriptive (particularly for when someone is deciding what yaml to look at). It's unclear from the name whether this is simpler use-cases of enrichers or simple yaml format for achieving the same thing with enrichers.

Assuming the latter, if there are old files with the old way, I'd prefer us to rename those yaml files to "legacy" or some such. If there aren't old files, then why call this "slightly-simpler"?